taxonomies/ingredients.txt

```diff
@@ -24566,6 +24566,7 @@ nl:bonenmeel
 <en:flour
 <en:flax seed
 en:flax floor, flaxseed floor
```

Comment:
flour?

Reply:
Not mine..but corrected as obviously wrong